src/styles/Login.scss

```diff
@@ -107,14 +107,11 @@
 
 .login-form-input {
   appearance: none;
-  background-color: $theme-dark;
-  border: 0;
-  border-left: 0;
+  border-color: #949494;
```

Comment:
Is this color used anywhere else? Ideally shouldn't have hardcoded colors in specific files.